conf/config-schema.json

```diff
@@ -0,0 +1,16 @@
+{
+    "type": "object",
+    "properties": {
+        "root": { "type": "boolean" },
+        "globals": { "type": ["object", "array"] },
+        "global": { "type": ["object", "array"] },
```

Comment:
Hmm, but I'm sure `.eslintrc` files does not support `global` property and the array value of `globals` property. Those are not working.

----

```
~\Documents\GitHub\sandbox2> cat .eslintrc.json
{
    "global": {"foo": false},
    "extends": "eslint:recommended"
}
~\Documents\GitHub\sandbox2> cat test.js
Object
foo
~\Documents\GitHub\sandbox2> eslint test.js

C:\Users\t-nagashima.AD\Documents\GitHub\sandbox2\test.js
  2:1  error  'foo' is not defined  no-undef

✖ 1 problem (1 error, 0 warnings)
```

----

```
~\Documents\GitHub\sandbox2> cat .eslintrc.json
{
    "globals": ["foo"],
    "extends": "eslint:recommended"
}
~\Documents\GitHub\sandbox2> cat test.js
Object
foo
~\Documents\GitHub\sandbox2> eslint test.js

C:\Users\t-nagashima.AD\Documents\GitHub\sandbox2\test.js
  2:1  error  'foo' is not defined  no-undef

✖ 1 problem (1 error, 0 warnings)
```

----

```
~\Documents\GitHub\sandbox2> cat .eslintrc.json
{
    "globals": {"foo": false},
    "extends": "eslint:recommended"
}
~\Documents\GitHub\sandbox2> cat test.js
Object
foo
~\Documents\GitHub\sandbox2> eslint test.js
```